buster/completers/base.py

```diff
@@ -23,7 +23,7 @@
 
 @dataclass(slots=True)
 class Completion:
-    completor: Iterator  # e.g. a response from openai.ChatCompletion
+    _completor: Iterator | str  # e.g. a streamed response from openai.ChatCompletion
     error: bool
```

Comment:
There is an error field in both `Completion` and `BusterAnswer`. How are they different? Maybe add a comment to specify what kind of error they track.

Reply:
You're right, it was duplicated, the one to be used is in Completion, not in `BusterAnswer`, fixed